test/distributed/_shard/sharded_tensor/test_sharded_tensor.py

```diff
@@ -2928,24 +2946,25 @@ def test_init_from_local_shards_and_global_metadata(self):
             self.assertEqual(1, len(shards))
             for remote_shard in shards:
                 self.assertEqual(rpc_rank, remote_shard.owner().id)
-                shard = remote_shard.to_here()
-                self.assertEqual((5, 5), shard.tensor.size())
+                if DEVICE_TYPE != "xpu":
```

Comment:
It is not reasonable to add specific logic in UT. Let's skip this ut and attach the corresponding torch-xpu-ops issue.